spinalcordtoolbox/utils/sys.py

```diff
@@ -381,12 +381,23 @@ def sct_dir_local_path(*args):
 
 
 def sct_test_path(*args):
-    """Construct a directory path relative to the sct testing data. Consults the
-    SCT_TESTING_DATA environment variable, if unset, paths are relative to the
-    current directory."""
+    """Construct a directory path relative to the sct testing data. Consults
+    the SCT_TESTING_DATA environment variable.
+    If unset, use the default dataset location from sct_download_data."""
 
     test_path = os.environ.get('SCT_TESTING_DATA', '')
-    return os.path.join(test_path, 'sct_testing_data', *args)
+    if test_path:
+        return os.path.join(test_path, 'sct_testing_data', *args)
```

Comment:
Now that I think about it, I feel a little skeptical about whether this `SCT_TESTING_DATA` environment variable is actually used at all?

It was added in #2804 to address @jcohenadad's https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/2804#discussion_r458225465:

> actually, when running pytest on my local station, i always do it in a temp folder (outside of SCT dir), so this will be a problem. Ideally we should set an env variable for where the testing data are downloaded

Have you used the `SCT_TESTING_DATA` environment variable at all in the last 1.5 years, @jcohenadad?

Reply:
(If not, it might be worth removing during a follow-up PR when we address https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3422.)

Reply:
I think I'm going to merge the changes as-is, and re-raise this point whenever we create a follow-up PR that makes `sct_test_path` into a pytest fixture.